### DISCUSSION TITLE
Getting started with development

### DISCUSSION THREAD

**Discussion by pholm** [hackjunction/hackplatform#440] [Ideas]
Currently starting development of Junction App is quite hard, as one has to first populate the database with test data and create super user to Auth0.

Most open-source projects have created initializing scripts for new collaborators. 

The script should do the following:
1. install all dependencies
2. fetch test data to local db
3. create new user to Auth0 with highest privileges

Thoughts?

**Comment by khenriksson** [comment]
I think this is an excellent suggestion, this is for example a problem I've had lately, especially with the fact that I'm not able to publish events, and therefore cannot test them 😄
